config.js

```diff
@@ -11,12 +11,142 @@ System.config({
 System.config({
```

Comment:
Hmm, doesn't I suppose. WF doesn't have any transitive dependencies, its flat I think. Must be why it works for that.

Q: is `config.js` read from a dependency when a package is loaded as a dependency?